test/correctness/mul_div_mod.cpp

```diff
@@ -315,7 +315,7 @@ bool mul(int vector_width, ScheduleVariant scheduling, const Target &target) {
             T1 ai = a(i, j);
             T2 bi = b(i, j);
             RT ri = r(i, j);
-            RT correct = RT(ai)*RT(bi);
+            RT correct = BIG(ai)*BIG(bi);
```

Comment:
Was this a fix or just a clarification? I think this change should be a no-op, but I agree your proposed change is more clear.

Reply:
If it is a clarification, should it also have an explicit cast to RT instead of an implicit one?

Reply:
It's a fix; there was one test case that would not pass UBSan without it. I can dig up the specific instance if you like.